ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,15 @@
-### Description of feature or bug
+#### Description of feature or bug
 
 TODO
 
-### Definition of done
+#### Definition of done
 
 TODO
 
--------
 _After evaluating, edit this part:_
 
-### Level of effort
+##### Level of effort - (low/medium/high)
 
-TODO: e.g., low/medium/high
-
-### Implementation outline (if higher than a "low" effort):
+### Implementation outline (if higher than "low" effort):
```

Comment:
should this be level 4 as well? ie `####`

Reply:
My mistake! yes, will fix